src/createExplorer.js

```diff
@@ -173,7 +174,7 @@ class Explorer {
 
   loadPackageProp(filepath: string, content: string) {
     const parsedContent = loaders.loadJson(filepath, content);
-    const packagePropValue = parsedContent[this.config.packageProp];
+    const packagePropValue = get(parsedContent, this.config.packageProp);
```

Comment:
We're taking a risk here that current devs aren't using a period in `packageProp`, which might be fine.  Just wanted to point out the possible breaking change.

Reply:
True, I guess we could use

```js
parsedContent[this.config.packageProp] || get(parsedContent, this.config.packageProp);
```

to mitigate the breaking change?

Reply:
ha yeah I guess that's easy enough.  :thumbsup:

Reply:
So it seems `lodash.get` already handles this case 😄 

Check it out https://runkit.com/iamogbz/5c60e0e12da4920012e32cad

Reply:
oye - til I learned.  Thanks much for the quick code snippet.